common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -154,6 +164,32 @@ public void setTenantId(String tenantId) {
         mTenantId = tenantId;
     }
 
+    /**
+     * Determines the audience type (MSA, Common, AAD, or Unknown) from the given
+     * {@link AzureActiveDirectoryAuthority}.
+     *
+     * @param authority The authority from which the audience is derived.
+     * @return The audience type: "MSA", "Common", "AAD", or "Unknown".
+     */
+    public static String getAudienceFromAuthority( final AzureActiveDirectoryAuthority authority) {
```

Comment:
Let's return the `AudienceTelemetryConstant` directly from the method (rather than calling `.name` ).

The callers of the method can invoke .name as needed

Reply:
added